Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1

```diff
@@ -9,6 +9,7 @@ $ErrorActionPreference = 'Stop'
 
 $script:deploymentSettingsFiles = @()
 $script:deploymentSettingsOverrides = @{ }
+Import-Module -Force -Scope Global "$PSScriptRoot/../../../logistics/scripts/modules/path-resolver.psm1"
```

Comment:
wouldn't importing `path-resolver` here prevent you from setting the `$env:toolsPath` below?

Reply:
OK fixed it . Thanks